src/e3fp/__init__.py

```diff
@@ -1,4 +1,4 @@
 import importlib.metadata
 from .util import E3FPWarning, E3FPDeprecationWarning
 
-__version__ = importlib.metadata.version("e3fp")
+# __version__ = importlib.metadata.version("e3fp")
```

Comment:
Why is this line now commented out?

Reply:
Sorry, that was a leftover from local experiments, I fixed it